### DISCUSSION TITLE
A Keyboard shortcut to enable/disable Secure Keyboard Entry on MacOS

### DISCUSSION THREAD

**Discussion by jrouzierinverse** [ghostty-org/ghostty#4119] [Q&A] [answered]
It would be nice to be able to toggle Secure Keyboard Entry on and off on MacOS via a shortcut

**Comment by mitchellh** [comment, marked as answer]
https://ghostty.org/docs/config/keybind/reference#toggle_secure_input

**Reply by jrouzierinverse** [reply]
Thank you.
I did the folllowing. 

`keybind = global:super+s=toggle_secure_input`

This was the only action in Ghostty drop down that did not have a default keyboard shortcut.
Should there be a default shortcut for this?
Or since this is only a Mac thing you decided against it.
